test/uk/gov/hmrc/apidefinition/service/AwsAPIPublisherSpec.scala

```diff
@@ -85,8 +85,8 @@ class AwsAPIPublisherSpec extends AsyncHmrcSpec {
       val swaggerDetailsCaptor: ArgumentCaptor[AWSSwaggerDetails] = ArgumentCaptor.forClass(classOf[AWSSwaggerDetails])
       when(underTest.awsAPIPublisherConnector.createOrUpdateAPI(*, swaggerDetailsCaptor.capture())(*))
         .thenReturn(successful(UUID.randomUUID().toString))
-      val apiDefinition1: ApiDefinition                           = someAPIDefinition("API 1")
-      val apiDefinition2: ApiDefinition                           = someAPIDefinition("API 2")
+      val apiDefinition1: StoredApiDefinition                     = someAPIDefinition("API 1")
```

Comment:
Perhaps these variables could change to `storedApiDefinition1` and `...2`?

Reply:
We think they are OK as they are